lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb

```diff
@@ -1053,7 +1053,7 @@ def cmd_upload(*args)
     src_items << last if src_items.empty?
 
     if args.size == 1
-      dest = last.split(/(\/|\\)/).last
```

Comment:
This is getting everything after the last occurrence of either a forward or backslash. I'm pretty sure the intention is to get the basename of the file from the path, so let's just call that method so it's more readable and accurate based on what the separator actually is.